tools/db/build/common.php

```diff
@@ -21,4 +21,25 @@ function cache($url, $filename, $duration = 60 * 60 * 24 * 7, $do_force = false)
     }
     return true;
   }  
+
+  /*
+   Recursively delete files from a folder plus the folder itself
+  */
+  function deleteDir($dirPath) {
+    if (!is_dir($dirPath)) {
```

Comment:
Thoughts on adding safeguards for `$dirPath`?
e.g. to avoid someone calling `deleteDir('/')

Reply:
Given we can already call `unlink('anything')` I think it's an unnecessary safeguard in a utility API :)